### ISSUE TITLE
The formatter replaces "test tags" always with deprecated "force tags" in Settings Section (RF 6.0.2)

### ISSUE THREAD

**Issue by phermann-DGL** [robocorp/robotframework-lsp#929]
**Describe the bug**
The formater in VSCode always replaces the **Test Tags** setting with the deprecated **Force Tags** setting.

**To Reproduce**
1. Create a new Robocorp Code Robot Framework Standard project
2. Open conda.yaml
3. Replace python=3.9.13 with python=3.11.1
4. Replace rpaframework==22.5.3 with robotframework==6.0.2
5. Save the file
conda.yaml
```
# For more details on the format and content:
# https://github.com/robocorp/rcc/blob/master/docs/recipes.md#what-is-in-condayaml
# Tip: Adding a link to the release notes of the packages helps maintenance and security.

channels:
  - conda-forge

dependencies:
  # Define conda-forge packages here -> https://anaconda.org/search
  # When available, prefer the conda-forge packages over pip as installations are more efficient.
  - python=3.11.1 \# https://pyreadiness.org/3.9/
  - pip=22.1.2 \# https://pip.pypa.io/en/stable/news/
  - pip:
      # Define pip packages here -> https://pypi.org/
      - robotframework==6.0.2
```

6. Open the tasks.robot file -> _Wait until Project Setup is done._
7. Rename *** Tasks *** to *** Test Cases *** -> __Note__: _With *** Tasks *** it will not replaced._
8. Add **Test Tags  example** entry to the settings section.
tasks.robot
```
*** Settings ***
Documentation       Template robot main suite.

Test Tags          example


*** Test Cases ***
Minimal task
    Log    Done.
```
10. Format the code

**Expected behavior**
Test Tags should NEVER replaced with Force Tags.

**Screenshots**
before format:
![image](https://github.com/robocorp/robotframework-lsp/assets/98079322/6a0cb306-f885-433b-9a4b-37aff982c65e)
after format:
![image](https://github.com/robocorp/robotframework-lsp/assets/98079322/08cbf8f8-ec63-4a42-b1b8-fb2266c273c8)

**Versions:**
 - OS: Windows 10 (64 Bit) [22H2 Build 19045-2843]
 - Robot Framework Version: 6.0.2
![image](https://github.com/robocorp/robotframework-lsp/assets/98079322/d3a1666a-5efd-4ee4-a621-849ac34a64f8)
 - Robot Framework Language Server Version: 1.10.1
 - Client Version: VSCode 1.78.1
 - Robocop Version: 3.1.1
 - RoboTidy Version: 4.2.1
 - python Version: 3.11.1

**Logs**

[logs.zip](https://github.com/robocorp/robotframework-lsp/files/11449907/logs.zip)

**Comment by bhirsz** [comment]
Hi, this looks like one of the old bugs in the Robotidy (I'm author of Robotidy btw, no connections to LSP, noticed your issue by chance). I've tested it locally with 4.2.1 and I can confirm it works with Robotidy. Maybe LSP is using built-in robotidy instead of your installed Robotidy? (they are keeping some of the libraries vendored in, see https://github.com/robocorp/robotframework-lsp/tree/master/robocorp-python-ls-core/src/robocorp_ls_core/libs).

Currently LSP vendores Robotidy 3.2 and your issue was fixed in 3.5.0 (https://github.com/MarketSquare/robotframework-tidy/issues/469)
